Enhanced the Dialog component based on code section

ARYANSHAH1567 opened the pull request:
**Description**

This PR fixes #6714

**Notes for Reviewers**
I have updated the overview and guidance section for the dialog component based on the current code section for it
url: /projects/sistent/components/dialog

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.
 

<!--
Thank you for contributing to Layer5 projects! 

Contributing Conventions:

1. Include descriptive PR titles with [<component-name>] prepended.
2. Build and test your changes before submitting a PR. 
3. Sign your commits

By following the community's contribution conventions upfront, the review process will 
be accelerated and your PR merged more quickly.
-->

ARYANSHAH1567 commented:
@vr-varad Can you please review this PR, Thank you!

yash37158 commented:
Hi @ARYANSHAH1567, I would recommend styling “Open Dialog” to look like a button instead of plain text.  also under the guidance are we really looking for Full-Screen Dialog ?

l5io commented:
🚀 Preview for commit 858167794a7fd8fa71624b45973550eb7bead7fc at: https://68a6ad0e2cb2f3bb0238b370--layer5.netlify.app

l5io commented:
🚀 Preview for commit c82e09d17db2938415d76b35b0c423d13684c2f6 at: https://68abf820942c222c0d8aed21--layer5.netlify.app

ARYANSHAH1567 commented:
<img width="1919" height="1024" alt="Screenshot 2025-08-26 141539" src="https://github.com/user-attachments/assets/2239b2d9-1cfb-4da5-aad5-7eca40bf5637" />

<img width="1919" height="1030" alt="Screenshot 2025-08-26 141522" src="https://github.com/user-attachments/assets/7c3ba04b-a202-480c-83eb-b34b80b0bb9d" />

@vr-varad I have changed the positioning of the button for guidance and code section, also in sistent navigation the entry for the dialog component was missing which was breaking the flow of navigation in components, so i added it too.
Can u please review this PR?

vr-varad commented on src/components/SistentNavigation/content.js:
```diff
@@ -38,49 +38,53 @@ export const content = [
+  { id: 30, link: "/projects/sistent/components/dialog/code", text: "Dialog" },
 
-  { id: 29, link: "/projects/sistent/components/link", text: "Link" },
-  { id: 30, link: "/projects/sistent/components/link/guidance", text: "Link" },
```
What are these changes about? @ARYANSHAH1567

ARYANSHAH1567 replied:
@vr-varad in this file, info regarding the dialog component was missing.
<img width="1919" height="1024" alt="image" src="https://github.com/user-attachments/assets/ddbd256f-f2f0-41d1-b826-6230c4515c2d" />

here it should show previous and next, but since the info is not present so it is not visible and if you click on next button it redirects to /sistent/about which should not happen

vr-varad replied:
ok @ARYANSHAH1567 got u!!

vr-varad commented:
@ARYANSHAH1567 There are merge conflict mind resolving it.
Otherwise it looks good to me.

l5io commented:
🚀 Preview for commit 9399faf2a193831c478d99df62fb1cad282764df at: https://68aedc8b2121a718198b4ff4--layer5.netlify.app

l5io commented:
🚀 Preview for commit 51ab919eb989262290b040bcfb2b2a5710c0dab7 at: https://68aff9625b8b195565372e6d--layer5.netlify.app
